src/site/apt/index.apt.vm

```diff
@@ -166,6 +166,29 @@ Welcome to ${project.name}!
 </project>
 +-----+
 
+** Matomo (i.e ApacheAnalytics)
+
+  Users can optionally enable <<Matomo>>.
+  How to configure it for Apache Analytics:
```

Comment:
This sample doesn't actually show how to configure for Apache Analytics, values aren't filled.